application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/RepairProperties.java

```diff
@@ -145,8 +156,24 @@ public static RepairProperties from(Properties properties) throws ConfigurationE
         long repairHistoryLookbackInMs = parseTimeUnitToMs(properties, CONFIG_REPAIR_HISTORY_LOOKBACK_BASE,
                 DEFAULT_REPAIR_HISTORY_LOOKBACK_DURATION, DEFAULT_REPAIR_HISTORY_LOOKBACK_TIMEUNIT);
 
+        long targetRepairSizeInBytes = Long.MAX_VALUE;
```

Comment:
Perhaps add 
`private static final long DEFAULT_TARGET_REPAIR_SIZE_IN_BYTES = Long.MAX_VALUE;`
like you have done in the test class and use it here.

Reply:
Done

Reply:
Or actually, I referenced RepairConfiguration.FULL_REPAIR_SIZE instead to make the default purpose clearer.